### PR TITLE
Set the status bar colour on mobile

### PR DESCRIPTION
**Pull request by SERVCUBED**
Set the status bar background colour to the splash screen background colour and set the foreground colour to white.

signal wp_ss_20171022_0001

### CODE REVIEW

File: Signal-Windows/Signal-Windows.csproj [hunk 1]
```diff
@@ -347,6 +347,11 @@
       <Version>2.2.9</Version>
     </PackageReference>
   </ItemGroup>
+  <ItemGroup>
+    <SDKReference Include="WindowsMobile, Version=10.0.16299.0">
```
Comment (golf1052) [inline Signal-Windows/Signal-Windows.csproj]: 16299 is FCU, we should probably stick to 15063 (CU) until we upgrade everything.

Comment (SERVCUBED) [reply Signal-Windows/Signal-Windows.csproj]: There isn't a mobile extension SDK version 15063 from what I can see. Version 16299 is backwards compatible down to UWP 10563. The 2nd latest version is 14393.

Comment (golf1052) [reply Signal-Windows/Signal-Windows.csproj]: Ah OK. Yeah I'm checking on my install as well and I can't find it but it looks like it should exist?
https://social.msdn.microsoft.com/Forums/windowsapps/en-US/64285e56-9522-43ec-9c29-a9e1c4b5fb61/uwprs21703problem-with-the-windows-10-creators-update?forum=wpdevelop

Comment (SERVCUBED) [reply Signal-Windows/Signal-Windows.csproj]: I'm thinking that maybe it was removed when the backwards compatible FCU version became available.

**Pull request merged**